{{cookiecutter.project_slug}}/tools/make/Common.mk

```diff
@@ -0,0 +1,57 @@
+# use sane shell by default, as well as other settings
+SHELL := bash
+.SHELLFLAGS := -eu -o pipefail -c
+.DELETE_ON_ERROR:
+MAKEFLAGS += --warn-undefined-variables --no-builtin-rules
+.DEFAULT_GOAL = help
+
+# location for source files
+SRC ?= src/
+
+# Hack to make help rule work, given that rule has suffix ' ## <help text'.
+# Minor adjustments to make it work properly with included files
+.PHONY: help
+help: ## This help dialog
+	@grep -hE '^[a-zA-Z-][a-zA-Z0-9_-]+:.*?## .*$$' $(MAKEFILE_LIST) | sort -u -t: -k1,1 | awk 'BEGIN {FS = ":.*?## "}; {printf "\033[36m%-22s\033[0m %s\n", $$1, $$2}'
+
+.PHONY: grep-tech-debt
+grep-tech-debt:  ## Grep for lines with various types of tech debt
+	@echo "### ❗Checking for FIXMEs"
+	@git grep --line-number --ignore-case --extended-regexp '# (fixme|xxx)' $(SRC) || echo "No FIXMEs ✅"
+	@echo "### 🚧 Silenced issues"
+	@git grep --line-number --ignore-case --extended-regexp '# (noqa|type: ignore|xxx)' $(SRC) || echo "No silenced issues ✅"
+	@echo "### 🚧 TODOs"
+	@git grep --line-number --ignore-case --extended-regexp '# (todo|tbd)' $(SRC) || echo "No TODOs ✅"
+
+######################################################################################################
+# Common rules which probably should be present regardless of technologies, but implemented
+# in specific files or root Makefile. Avoid modifying this file, because it would be updated by the template
+#
+# Note that specifying .PHONY here means actual implementation
+# needs only rule body (repeating .PHONY is not needed)
+######################################################################################################
+
+.PHONY: deps
```

Comment:
Another option would be to comment these by default, so that rules that are not implemented are not visible with `make help`